src/obs_inv_utils/search_engine.py

```diff
@@ -176,6 +194,46 @@ def parse_filename(filename):
     return filename_meta
 
 
+def process_aws_s3_list_objects_v2_resp(contents):
```

Comment:
same question about "v2"

Reply:
This is a suggestion: Should hpss and aws specific functionality reside in hpss and aws specific modules and the search engine should be oblivious if hpss or aws is used?

Reply:
That is my goal, yes.  I just wanted to produce something quickly to scan/spider the aws inventory.  I can create a new pull request (dependent on this one) which makes search engine agnostic.

Reply:
Maybe make an issue and address it in a separate PR after this one is merged?